tests/orm/test_querybuilder.py

```diff
@@ -852,6 +852,58 @@ def test_empty_filters(self):
         qb = orm.QueryBuilder().append(orm.Data, filters={'or': [{}, {}]})
         assert qb.count() == count
 
+    @pytest.mark.usefixtures('aiida_profile_clean')
+    def test_abstract_code_filtering(self, tmp_path):
+        """Test that querying for AbstractCode correctly returns all code instances.
+
+        This tests the fix for issue #6687, where QueryBuilder couldn't find codes
+        when looking for AbstractCode due to a node_type mismatch.
+        """
+        # Set up test environment
+
+        computer = orm.Computer(
```

Comment:
We should use existing test fixtures here instead of managing manually, e.g. `aiida_code` or `aiida_code_installed`.